Assignment_1/scikit.py

```diff
@@ -0,0 +1,17 @@
+import numpy as np
+from sklearn.neighbors import KNeighborsClassifier
+np.set_printoptions(precision=2)
+ftrain = open("TrainingData_A1.tsv")
+ftrain.readline()  # skip the header
+data = np.loadtxt(ftrain)
+Xtrain = data[:, 0:data.shape[1]-1]  
+ytrain = data[:, data.shape[1]-1] 
+ftest= open("TestData_A1.tsv")
```

Comment:
I think it is better to add one more space before the equal sign just as the previous code style.
for more python style guide tips, you can check PEP 8: https://realpython.com/python-pep8/